provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -94,11 +70,49 @@ public PractitionerController(
             Practitioner.Bundle::new));
   }
 
+  /** If a user were to search by a parameter other then family or given, the call would
+   * be checked and failed earlier at the PPMS call. */
```

Comment:
Method javadoc should describe the behavior of the method as a whole. This note, about why it's safe to assume that the `family` and `given` parameters are present if `identifier` and `name` are not, should be an inline comment down in that `else` block.